common/build.gradle

```diff
@@ -28,7 +28,7 @@ codeCoverageReport {
     coverage.enabled = enableCodeCoverage
 }
 
-def common4jVersion = "0.0.+"
+def common4jVersion = "2.0.0"
```

Comment:
should we restore this value?

Reply:
We should definitely restore this. Good catch @p3dr0rv